services/src/unit-test/java/uk/gov/moj/sdt/services/SubmitQueryServiceTest.java

```diff
@@ -259,121 +234,101 @@ public void testSubmitQueryRequestSoapFault() {
         receiveTimeOutParam.setValue("12000");
-        EasyMock.expect(this.mockGlobalParamCache.getValue(IGlobalParameter.class, "TARGET_APP_RESP_TIMEOUT"))
-                .andReturn(receiveTimeOutParam);
+        when(mockGlobalParamCache.getValue(IGlobalParameter.class, TARGET_APP_RESP_TIMEOUT))
+            .thenReturn(receiveTimeOutParam);
 
         final IGlobalParameter maxQueryReq = new GlobalParameter();
         maxQueryReq.setName("MCOL_MAX_CONCURRENT_QUERY_REQ");
         maxQueryReq.setValue("5");
-        EasyMock.expect(this.mockGlobalParamCache.getValue(IGlobalParameter.class, "MCOL_MAX_CONCURRENT_QUERY_REQ"))
-                .andReturn(maxQueryReq);
+        when(mockGlobalParamCache.getValue(IGlobalParameter.class, "MCOL_MAX_CONCURRENT_QUERY_REQ"))
+                .thenReturn(maxQueryReq);
 
         final SoapFaultException soapFaultEx = new SoapFaultException("SOAPFAULT_ERROR", "Soap fault occurred");
-        this.mockConsumerGateway.submitQuery(submitQueryRequest, 1000, 12000);
-        EasyMock.expectLastCall().andThrow(soapFaultEx);
+
+        doThrow(soapFaultEx).when(mockConsumerGateway).submitQuery(submitQueryRequest, 1000, 12000);
 
         final IErrorMessage errorMsg = new ErrorMessage();
         errorMsg.setErrorCode("SDT_INT_ERR");
         errorMsg.setErrorDescription("SDT Internal Error.");
         errorMsg.setErrorText("SDT Internal Error, please report to {0}");
-        EasyMock.expect(this.mockErrorMsgCacheable.getValue(IErrorMessage.class, "SDT_INT_ERR")).andReturn(errorMsg);
+        when(mockErrorMsgCacheable.getValue(IErrorMessage.class, "SDT_INT_ERR")).thenReturn(errorMsg);
 
         final IGlobalParameter contactNameParameter = new GlobalParameter();
         contactNameParameter.setValue("Tester");
         contactNameParameter.setName("CONTACT_DETAILS");
-        EasyMock.expect(this.mockGlobalParamCache.getValue(IGlobalParameter.class, "CONTACT_DETAILS")).andReturn(
+        when(mockGlobalParamCache.getValue(IGlobalParameter.class, "CONTACT_DETAILS")).thenReturn(
                 contactNameParameter);
 
-        EasyMock.replay(mockConsumerGateway);
-        EasyMock.replay(mockGlobalParamCache);
-        EasyMock.replay(mockErrorMsgCacheable);
-        EasyMock.replay(mockBulkCustomerDao);
-
         SdtContext.getContext().setRawInXml("criteria");
         this.submitQueryService.submitQuery(submitQueryRequest);
 
-        EasyMock.verify(mockConsumerGateway);
-        EasyMock.verify(mockGlobalParamCache);
-        EasyMock.verify(mockErrorMsgCacheable);
-        EasyMock.verify(mockBulkCustomerDao);
+        assertNull(SdtContext.getContext().getRawOutXml(),"Raw output xml should be null");
 
-        Assert.assertEquals("Raw output xml should be null", null, SdtContext.getContext().getRawOutXml());
+        verify(mockConsumerGateway).submitQuery(submitQueryRequest, 1000, 12000);
+        assertTrue(true,"Expected to pass");
```

Comment:
This doesn't seem to serve a purpose, especially as there's a check that occurs after it.  Suggest removing it or moving it to the end of the test to be consistent with the other tests.

Reply:
done